storybook/src/pages/amsterdam/ArticlePage/ArticleBody.tsx

```diff
@@ -63,7 +63,7 @@ export const ArticleBody = ({ lead, paragraph1, spotlightHeading, spotlightLinkL
       <Grid.Cell span={{ narrow: 4, medium: 5, wide: 7 }} start={{ narrow: 1, medium: 2, wide: 3 }}>
         <Column>
```

Comment:
Space between a Paragraph and a Heading 2 should be x-large.

```suggestion
        <Column gap="x-large">
```